ibis/common/grounds.py

```diff
@@ -244,3 +260,14 @@ def args(self):
     @property
     def argnames(self):
         return self.__argnames__
+
+    def map(self, fn):
```

Comment:
Looks like this method could use a test or two.

Reply:
Moved the comment to the other PR